ui/apos/components/AposExportModal.vue

```diff
@@ -70,42 +70,37 @@
           >
             <div
               v-show="!relatedDocumentsDisabled"
-              class="apos-export__section"
+              class="apos-export__section apos-export__section--related-documents"
             >
-              <div
```

Comment:
remove unnecessary container, as well as its ref because we no longer need/use it